packages/pyright-internal/src/languageService/hoverProvider.ts

```diff
@@ -142,6 +144,12 @@ export class HoverProvider {
             if (type !== undefined) {
                 this._tryAddPartsForTypedDictKey(format, sourceMapper, evaluator, node, type, results.parts);
             }
+        } else if (node.nodeType === ParseNodeType.ModuleName || node.nodeType === ParseNodeType.Module) {
+            const importInfo = getImportInfo(node);
+            if (importInfo) {
+                this._addResultsPart(results.parts, '(module) ' + importInfo.importName, /* python */ true);
```

Comment:
also, we have been trying to match tooltip behavior between completion provider and hover, so it would be nice if we can share these with completion provider as well.

Reply:
The completion provider should provide doc strings as I type?

Like this would show them on the right?

![image](https://user-images.githubusercontent.com/19672699/205769414-7e593125-6e33-40a3-b55e-30d3c6c8149c.png)

Reply:
Can I enter another issue to do that? Seems like a bigger issue.

Reply:
ya, let me open an issue. we do provide doc string for other cases, it looks like somehow we don't do it for completion.